lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
The AJAX calls would need to be changed to `js/spice/pubmed_abstract` as well, instead of `pubmed/abstract`